tests/OTAnalytics/domain/track_dataset/conftest.py

```diff
@@ -0,0 +1,156 @@
+from typing import Literal
+from unittest.mock import Mock
+
+import pytest
+
+from OTAnalytics.domain.track import Track
+from OTAnalytics.domain.track_dataset import (
+    TRACK_GEOMETRY_FACTORY,
+    FilteredTrackDataset,
+    TrackDataset,
+)
+from OTAnalytics.plugin_datastore.python_track_store import (
+    FilteredPythonTrackDataset,
+    PythonTrackDataset,
+)
+from OTAnalytics.plugin_datastore.track_geometry_store.pygeos_store import (
+    PygeosTrackGeometryDataset,
+)
+from OTAnalytics.plugin_datastore.track_store import (
+    FilteredPandasTrackDataset,
+    PandasTrackDataset,
+)
+from OTAnalytics.plugin_prototypes.track_visualization.track_viz import (
+    CLASS_BICYCLIST,
+    CLASS_CAR,
+    CLASS_CARGOBIKE,
+    CLASS_PEDESTRIAN,
+    CLASS_TRUCK,
+)
+from tests.conftest import create_track
+
+PYTHON: Literal["PYTHON"] = "PYTHON"
+PANDAS: Literal["PANDAS"] = "PANDAS"
+IMPLEMENTATIONS = [PYTHON, PANDAS]
+
+
+@pytest.fixture
+def car_track() -> Track:
+    return create_track("1", [(1, 1), (2, 2)], 1, CLASS_CAR)
+
+
+@pytest.fixture
+def car_track_continuing() -> Track:
+    return create_track("1", [(3, 3), (4, 4), (5, 5)], 3, CLASS_TRUCK)
+
+
+@pytest.fixture
+def pedestrian_track() -> Track:
+    return create_track("2", [(1, 1), (2, 2), (3, 3)], 1, CLASS_PEDESTRIAN)
+
+
+@pytest.fixture
+def bicycle_track() -> Track:
+    return create_track("3", [(1, 1), (2, 2), (3, 3)], 4, CLASS_BICYCLIST)
+
+
+@pytest.fixture
+def cargo_bike_track() -> Track:
+    return create_track("4", [(1, 1), (2, 2), (3, 3)], 4, CLASS_CARGOBIKE)
```

Comment:
That sounds good and reasonable. I will move the fixtures to `tests/conftest.py`.

 It might be also a good idea to move all convenience classes and functions to a separate module or structure. `conftest` files would then only hold fixtures.

Reply:
That sounds reasonable. 👍